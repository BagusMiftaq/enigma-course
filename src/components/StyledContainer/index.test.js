import {render} from "@testing-library/react";

import StyledContainer from "./index";

test("should render StyledContainer correctly",()=>{
    const { asFragment } = render(<StyledContainer/>);

    expect(asFragment()).toMatchSnapshot();
})