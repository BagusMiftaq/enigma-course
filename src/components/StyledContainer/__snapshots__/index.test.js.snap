// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render StyledContainer correctly 1`] = `
<DocumentFragment>
  <div
    class="sc-bcXHqe heOmnA container"
  />
</DocumentFragment>
`;
